AdyenActions/Components/QRCode/QRCodeActionComponent.swift

```diff
@@ -175,12 +185,12 @@ public final class PixActionComponent: ActionComponent, Cancellable {
     
     fileprivate func cleanup() {
         timeoutTimer?.stopTimer()
-        pollingComponent?.didCancel()
```

Comment:
this is still needed for PIX, right? why is this removed?

Reply:
Added this back 😄